version.json

```diff
@@ -0,0 +1,11 @@
+{
+  "version": "2.4",
+  "publicReleaseRefSpec": [
+    "^refs/heads/v2", // we release out of v2
+    "^refs/tags/v\\d+\\.\\d+", // we also release tags starting with vN.N
+    "^refs/heads/rel/v\\d+\\.\\d+" // Also release branches starting with rel/vN.N
```

Comment:
This should change to:

`^refs/tags/v2\\.\\d+\\.\\d+`
`^refs/heads/rel/v2\\.\\d+\\.\\d+`

It not only makes it clear that these will only be used for v2 tags/branches (clarity) but also we use 3 digit versions, not 2.